python/social_bot/teacher_tasks.py

```diff
@@ -274,10 +276,6 @@ def __init__(self,
             percent_full_range_in_curriculum=percent_full_range_in_curriculum,
             max_reward_q_length=max_reward_q_length)
         self._random_goal = random_goal
-        self._objects_in_world = [
-            'placing_table', 'plastic_cup_on_table', 'coke_can_on_table',
-            'hammer_on_table', 'cafe_table', 'ball'
-        ]
```

Comment:
Is this deletion intentional?

Reply:
yes, this makes the task depends on world file, besides
 the default world file has changed to playground.world, so removed

Reply:
I see.  This changes the setup of the task though, so at least indicate in the commit message that this is happening.

Reply:
OK, I'll add the message when checking in.